docs/python-examples/debug-spaces.py

```
@@ -0,0 +1,116 @@
+# Debug script to check what spatial elements exist in the IFC file
+# Use this in a Python node to understand the IFC structure
+
+import ifcopenshell
+
+# The IFC model is provided as 'ifc_file' by the system
+model = ifc_file
```

Ruff (0.12.2)

Undefined name `ifc_file`

(F821)